cmd_version.lua

```diff
@@ -0,0 +1,13 @@
+function HandleVersionCommand(Split, Player)
+	-- Send output to player:
+	SendMessage(Player, string.format(
+		"This server is running Cuberite %s - %s (%s - %s) with Lua version %s and SQL version %s.",
```

Comment:
It's not `SQL` as in the generic sense, it's `SQLite`, a specific library providing SQL-like capabilities on local files. Not sure it warrants reporting its version, considering we don't update too much.